src/Select.js

```diff
@@ -599,9 +600,7 @@ export default class Select extends Component<Props, State> {
     }
     if (this.props.isDisabled) return;
     const { isMulti, menuIsOpen } = this.props;
-    if (!this.focused) {
-      this.focusInput();
-    }
+    this.focusInput();
```

Comment:
good catch! This afaik is a bug, but I'll let @JedWatson  confirm this. but ideally we should still be doing a check for whether or not the input is focused, the conditional should be `if (!this.state.isFocused)` 

Once again, I'll let either @JedWatson  or @jossmac  confirm this

Reply:
I'll have to look at this tomorrow - we _were_ tracking the instance property because `setState` is async, and unreliable for this. So the solution may be to ensure that is re-implemented.

Reply:
Actually, got a chance to look through and I believe this is safe to do without the check. So going to merge 👍